pom.xml

```diff
@@ -161,9 +303,22 @@
             <name>org-sagebridge-repo-maven-releases</name>
             <url>https://repo-maven.sagebridge.org/</url>
         </repository>
+        <repository>
+            <id>sagebionetworks-libs-releases-local</id>
+            <name>sagebionetworks-libs-releases-local</name>
+            <url>http://sagebionetworks.artifactoryonline.com/sagebionetworks/libs-releases-local</url>
+        </repository>
     </repositories>
 
     <build>
+        <extensions>
+            <extension>
+                <groupId>org.springframework.build</groupId>
+                <artifactId>aws-maven</artifactId>
+                <version>5.0.0.RELEASE</version>
+            </extension>
+        </extensions>
```

Comment:
I believe this is used to publish to a Maven repo, which is not necessary.

Reply:
removed